modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -13,6 +13,263 @@
 
 toc::[]
 
+== Infinite Scale 5.0.0
+[discrete]
+=== General
+
+This release brings significant enhancements in performance and stability, particularly for deployments based on Kubernetes. It is also designed to provide an optimized experience for both end users and administrators.
+Please find the most important changes here described or refer to the changelog for a complete list of changes:
+
+* Infinite Scale: https://github.com/owncloud/ocis/releases/tag/v5.0.0[Changes in 5.0.0]
+* Web: https://github.com/owncloud/web/releases/tag/v8.0.0[Changelog for ownCloud Web 7.1.0]
+
+[discrete]
+=== Federated Cloud Sharing
+
+Federated Cloud Sharing allows users to access and collaborate on files stored on connected external servers, as if they were stored on the user's instance. This creates a fluid and efficient workflow, eliminating the need for multiple file versions or complex file transfer processes. With Federated Cloud Sharing data can reside on its original server, ensuring compliance with regional data sovereignty laws and organizational data policies.
+
+To enable file sharing with a user on a different, connected server, an initial invitation needs to be sent. Once this is completed, files can be shared with that user as usual by entering their username, just like with internal users. The only difference is that users from connected instances will be marked as "Federated Users" for easy identification and management.
+
+*Technical Note:* Federated Cloud Sharing in Infinite Scale is based on the Open Cloud Mesh protocol version 1.1 (OCM 1.1) https://cs3org.github.io/OCM-API/docs.html?branch=v1.1.0&repo=OCM-API&user=cs3org#/paths/~1shares/post[Learn more about OCM 1.1]
+
+Kudos to all members of the CS3-Community who contributed to this project. A special thanks is owed to CERN for their contribution of the Science Mesh App. Thank You!
+
+[discrete]
+=== Password Policy for Sharing Links
```

Comment:
Should we group the password policy and the default link permission closer together in the release notes?

Reply:
yes, both items are security related. makes sense.